Sources/Workspace/Workspace.swift

```diff
@@ -106,45 +91,6 @@ public protocol WorkspaceDelegate: AnyObject {
     func didDownloadBinaryArtifacts()
 }
 
-public extension WorkspaceDelegate {
```

Comment:
this is likely to impact users of libSwiftPM, but I think worth doing o that the delegates dont run into runtime issues when we deprecate APIs

Reply:
Agreed, this is worth resolving in the client.  Once we adopt semantic versioning this will also stop becoming an issue.